src/components/FeatureStyler/FeatureStyler.js

```diff
@@ -497,6 +591,30 @@ class FeatureStyler extends PureComponent {
       textRotation = featStyle.getText().getRotation();
     }
 
+    const oldStyles = FeatureStyler.getStyleAsArray(feature);
+
+    if (oldStyles.length > 1) {
```

Comment:
This 'if' seems useless

Reply:
If there aren't any extra styles defined, oldStyles has only a length of 1. Then I can't start iterating with 1.

Reply:
if oldStyles.length <= 1 , the code just doesn't go inside the loop    because i < arr.length

Reply:
fixed